src/applications/disability-benefits/686/config/form.js

```diff
@@ -157,122 +422,18 @@ const formConfig = {
                 'ui:title': 'Your suffix',
               },
             }),
-            'view:relationshipToVet': {
-              'ui:title': 'Your relationship to the Veteran',
-              'ui:widget': 'radio',
-              'ui:options': {
-                labels: {
-                  1: 'I am the Veteran',
-                  2: 'Spouse or surviving spouse',
-                  3: 'Unmarried adult child',
-                  4: 'Other',
-                },
-                nestedContent: {
-                  2: spouseRelationshipDescription,
-                  3: childRelationshipDescription,
-                  4: otherRelationshipDescription,
-                },
-              },
-            },
-          },
-          schema: {
-            type: 'object',
-            required: ['claimantFullName', 'view:relationshipToVet'],
-            properties: {
-              claimantFullName,
-              'view:relationshipToVet': {
-                type: 'string',
-                enum: ['1', '2', '3', '4'],
-              },
-            },
-          },
-        },
-        claimantInformation: {
-          path: 'claimant-information',
-          title: 'Applicant Information',
-          depends: formData => !isVeteran(formData),
-          uiSchema: {
-            claimantSocialSecurityNumber: _.merge(ssnUI, {
-              'ui:title': 'Your Social Security number',
-            }),
-            claimantAddress: address.uiSchema('Your address'),
-          },
-          schema: {
-            type: 'object',
-            required: ['claimantSocialSecurityNumber'],
-            properties: {
-              claimantSocialSecurityNumber,
-              claimantAddress: address.schema(fullSchema686, true),
-            },
-          },
-        },
-      },
-    },
-    veteranInformation: {
-      title: 'Veteran Information',
-      pages: {
-        veteranInformation: {
-          title: 'Veteran Information',
-          path: 'veteran-information',
-          uiSchema: {
-            veteranFullName: _.merge(fullNameUI, {
-              'ui:options': {
-                hideIf: formData => isVeteran(formData),
-              },
-              first: {
-                'ui:title': 'Veteran’s first name',
-                'ui:required': formData => !isVeteran(formData),
-              },
-              middle: { 'ui:title': 'Veteran’s middle name' },
-              last: {
-                'ui:title': 'Veteran’s last name',
-                'ui:required': formData => !isVeteran(formData),
-              },
-              suffix: { 'ui:title': 'Veteran’s suffix' },
-            }),
             veteranSocialSecurityNumber: _.merge(_.unset('ui:title', ssnUI), {
```

Comment:
Are we able to use `personId` from `vets-website/src/platform/forms/definitions/` here?